README.md

```diff
@@ -1,106 +1,184 @@
+
+Install the AWS CDK
+
+```
+npm install -g aws-cdk
+```
 
-## What We Care About
+Clone the repository
 
-Use any libraries that you would normally use if this were a real production App. Please note: we're interested in your code & the way you solve the problem, not how well you can use a particular library or feature.
+```
+git clone <this repository url>
+```
+
+Go to the repository's directory and install. This installs packages for the CDK stack and for the Lambda Layer.
+
+```
+cd <local repo directory>
+npm install
+```
+
+NOTE: If this is your first time running the AWS CDK in this AWS account and/or region, you need to boostrap the CDK
+
+```
+cdk boostrap
+```
 
-_We're interested in your method and how you approach the problem just as much as we're interested in the end result._
+Optional: Synthesize the stack to see the CloudFormation template the CDK generates
 
-Here's what you should strive for:
+```
+cdk synth
+```
 
-- Good use of current Node.js & API design best practices.
-- Solid testing approach.
-- Extensible code.
+Deploy the stack (note the various options)
 
-If you have not been specifically asked, you may pick either `Implementation Path: Docker Containers` or `Implementation Path: Cloud-native` requirements below.
+```
+// Example #1: standard 'DEV' stack
+cdk deploy
 
-## Implementation Path: Docker Containers
+// Example #2: deploys to a stack named 'TEST'
+cdk deploy -c stage=test
 
-### Basic Requirements
+// Example #3: deploys to a stack named 'PROD' with using a specific AWS profile.
+cdk deploy -c stage=prod --profile MY_PROFILE_NAME
+```
 
-  - Use Node.js `LTS` and any framework of your choice.
-  - Use any persistence store. NoSQL DB is preferred.
-  - Write concise and clear commit messages.
-  - Write clear **documentation** on how it has been designed and how to run the code.
+After a few minutes the stack should respond with the API endpoint
 
-### Bonus
+```
+// Example output (your endpoint URL will be different)
 
-  - Provide proper unit tests.
-  - Add a read only endpoint to fetch location information based off the user's address (use [NASA](https://api.nasa.gov/api.html) or [Mapbox](https://www.mapbox.com/api-documentation/) APIs)
-  - Use Docker containers.
-  - Utilize Docker Compose.
-  - Setup a CircleCI config to build/test/deploy the service.
-  - Leverage Terraform or other infrastructure management tooling to spin up needed resources.
-  - Providing an online demo is welcomed, but not required.
+// NOTE: The endpoint will not have the trailing '/users' which you will need to add when interacting the api.
 
-### Advanced Requirements
+Outputs:
+SF-UserStack-DEV.SFUserStackDEVUsersAPIEndpointC5532769 = https://epvz8ugbl7.execute-api.us-east-1.amazonaws.com/prod/
+```
 
-These may be used for further challenges. You can freely skip these if you are not asked to do them; feel free to try out if you feel up to it.
+## Create a User
```

Comment:
Thank you for your review, Afaq.  Yes, the ability to export was a primary reason behind defining the methods and validation within API Gateway instead of proxying and handling it within Lambda.